skaffold.yaml

```
@@ -20,7 +20,7 @@ requires:
       - etcd
 build:
   artifacts:
-    - image: eu.gcr.io/gardener-project/gardener/apiserver
+    - image: europe-docker.pkg.dev/gardener-project/snapshots/gardener/apiserver
```

```suggestion
    - image: europe-docker.pkg.dev/gardener-project/releases/gardener/apiserver
```
same here ([ref](https://github.com/gardener/gardener/pull/8968#issuecomment-1863415635))